tools/build-image.sh

```diff
@@ -85,7 +85,7 @@ if ! kiwi-ng --version &>/dev/null ; then
   error_exit "missing kiwi-ng"
 fi
 
-if ! sudo mkfs.btrfs --version &>/dev/null ; then
+if ! /sbin/mkfs.btrfs --version &>/dev/null ; then
   echo "error: missing btrfstools"
```

Comment:
nit: It was a typo when I committed this patch earlier, won't you mind to change 'btrfstools' to 'btrfsprogs package' too?